--- origsrc/libopensync-plugin-jescs-0.22/src/Makefile.am	2007-03-27 06:49:49.000000000 -0500
+++ src/libopensync-plugin-jescs-0.22/src/Makefile.am	2008-05-08 23:21:52.593750000 -0500
@@ -11,9 +11,7 @@
 
 plugin_LTLIBRARIES = jescs.la
 
-libexec_PROGRAMS = wcaptool
-
-wcaptool_SOURCES = wcaptool.pl
+libexec_SCRIPTS = wcaptool
 
 jescs_la_SOURCES = jescs.c
 jescs_la_CFLAGS = -DWCAPTOOL=\"$(libexecdir)/wcaptool\"
